tobler/tests/test_utils.py

```diff
@@ -57,8 +57,9 @@ def test_h3fy_clip_buffer():
     sac1 = geopandas.read_file(sac1.get_path("sacramentot2.shp"))
     sac_hex = h3fy(sac1, clip=True, buffer=True)
     sac_hex = sac_hex.to_crs(sac_hex.estimate_utm_crs())
+    sac1 = sac1.to_crs(sac_hex.estimate_utm_crs())
     assert_almost_equal(
-        sac_hex.area.sum(), 13749446323.1722, decimal=0
+        sac_hex.area.sum(), sac1.area.sum(), decimal=-8
```

Comment:
the decimal here is a bit drastic but it ensures that this works across different versions of h3 without if/else.